plugins/inputs/mdstat/mdstat.go

```diff
@@ -0,0 +1,288 @@
+package mdstat
+
+import (
+	"bufio"
+	//"fmt"
+	"errors"
+	"io"
+	"os"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+// #############################################################################
+// Telegraf plugin stuff
+// #############################################################################
+type MDSTAT_PLUGIN struct {
```

Comment:
Uppercase with underscore isn't go style.  Identifiers (of types, variables, consts, funcs, etc) should be camelCase, with initial letter captialized only if they need to be exported.

CI runs a linter after dependency checks pass.  I suspect the linter will not approve uppercase identifiers when it gets a chance to run.

Reply:
I have adjusted the identifiers to conform with the camelCase style.